### DISCUSSION TITLE
Is this library incomplete?

### DISCUSSION THREAD

**Discussion by babakfp** [markedjs/marked#2638] [General]
Hi
I realized this syntax doesn't work:
```
---
title: Hello, World!
---
```
So, is there a better Markdown parser?

**Comment by UziTech** [comment]
Should that syntax work? What are you expecting it to do?

**Reply by babakfp** [reply]
For example, you can write down your blog post title, publish date and etc at the top of the blog post markdown file and later in a template take that (title and etc) as a variable and show them in different places in the blog post singe page.

**Reply by UziTech** [reply]
That is called [frontmatter](https://www.npmjs.com/package/frontmatter) and it is not part of the markdown spec. I think you are looking for something like [jekyll](https://jekyllrb.com/).
